src/inscription.rs

```diff
@@ -713,6 +715,15 @@ mod tests {
   fn is_graphical() {
     assert!(inscription("image/png", []).is_graphical());
     assert!(!inscription("foo", []).is_graphical());
+    assert!(inscription("image/gif", []).is_graphical());
     assert!(!Inscription::new(None, Some(Vec::new())).is_graphical());
   }
+
+  #[test]
+  fn inscribe_gif() {
+    assert_eq!(
+      InscriptionParser::parse(&container(&[b"ord", &[1], b"image/gif", &[], &[1; 100]])),
+      Ok(inscription("image/gif", [1; 100])),
+    );
+  }
```

Comment:
I think this test can actually be removed, since inscription parsing didn't change.

```suggestion
```